recipe/custom_clang_toolchain/cc_toolchain_config.bzl

```diff
@@ -285,23 +293,29 @@ def _impl(ctx):
         ],
     )
 
-    cxx_builtin_include_directories = [
-        "${CONDA_BUILD_SYSROOT}/System/Library/Frameworks",
-        "${CONDA_BUILD_SYSROOT}/usr/include",
-        "${BUILD_PREFIX}/lib/clang/${CONDA_CLANG_VERSION}/include",
-        "${BUILD_PREFIX}/include/c++/v1",
-        "${PREFIX}/include",
-    ]
+    if "TARGET_PLATFORM".startswith("osx"):
+        cxx_builtin_include_directories = [
+            "${CONDA_BUILD_SYSROOT}/System/Library/Frameworks",
+            "${CONDA_BUILD_SYSROOT}/usr/include",
+            "${BUILD_PREFIX}/lib/clang/${CONDA_CLANG_VERSION}/include",
+            "${BUILD_PREFIX}/include/c++/v1",
+            "${PREFIX}/include",
+        ]
+    else:
+        cxx_builtin_include_directories = [
+            "${CONDA_BUILD_SYSROOT}/usr/include",
+	    "${BUILD_PREFIX}/lib/gcc/x86_64-conda-linux-gnu/9.3.0",
+	    "${BUILD_PREFIX}/x86_64-conda-linux-gnu/include/c++/9.3.0",
```

Comment:
Can `x86_64-conda-linux-gnu` also be made dynamic, based on `$HOST` ?
Can `9.3.0` also be made dynamic, based on the version of GCC being used?

Reply:
Good catch, I did forget to template that. This is also the root of the `ppc64le` failure (and would pop up as the next error in `aarch64` once I have fixed the build CPU there).